.pylintrc

```diff
@@ -70,7 +72,10 @@ confidence=
 # --enable=similarities". If you want to run only the classes checker, but have
 # no Warning level messages displayed, use"--disable=all --enable=classes
 # --disable=W"
-disable=E1608,W1627,E1601,E1603,E1602,E1605,E1604,E1607,E1606,W1621,W1620,W1623,W1622,W1625,W1624,W1609,W1608,W1607,W1606,W1605,W1604,W1603,W1602,W1601,W1639,W1640,I0021,W1638,I0020,W1618,W1619,W1630,W1626,W1637,W1634,W1635,W1610,W1611,W1612,W1613,W1614,W1615,W1616,W1617,W1632,W1633,W0704,W1628,W1629,W1636
+# disable=E1608,W1627,E1601,E1603,E1602,E1605,E1604,E1607,E1606,W1621,W1620,W1623,W1622,W1625,W1624,W1609,W1608,W1607,W1606,W1605,W1604,W1603,W1602,W1601,W1639,W1640,I0021,W1638,I0020,W1618,W1619,W1630,W1626,W1637,W1634,W1635,W1610,W1611,W1612,W1613,W1614,W1615,W1616,W1617,W1632,W1633,W0704,W1628,W1629,W1636
+# W0640: lambdas inside a loop being used to encapsulate functions for retry()
+# E1101: arcpy/arcgis nested calls (gis.users.me); normal errors caught appropriately by pylance
```

Comment:
**suggestion**: sometimes it is more appropriate to disable a check for a specific case instead of globally disabling it

Reply:
Agreed. In this case, however, pylance seems to also catch these errors, so I'd rather disable this than have to `# pylint: disable=no-member` everywhere. It seems cleaner, at least to me.

Reply:
And after reading up on W0640, it appears this will only happen on these lambda calls, so I made the similar judgement call on this one.

Reply:
Remember that pylance will not be available when the tests are running in CI. Therefore, you cannot rely on pylance for anything other than development productivity.

Reply:
I was thinking that pylint was also just a productivity tool. Do the CI tools also report pylint errors?

Reply:
Yes, since pytest runs all of the plugins, CI will fail on pylint errors.

Reply:
Ah, ok, did not know that. I'll re-evaluate.

Reply:
Should be fixed now.